src/ragas/experimental/testset/simulators/generate.py

```diff
@@ -80,21 +80,21 @@ def generate_with_langchain_docs(
     def generate(
         self,
         test_size: int,
-        scenarios: t.Optional[QuestionTypes] = None,
+        simulators: t.Optional[QuestionTypes] = None,
```

Comment:
Also here, if it's a simulator what is then QuestionTypes?

Reply:
changed this to `simulator_distributions` - how does that sound? is it clear?